packages/core/template_project/app/src/main/AndroidManifest.xml

```diff
@@ -47,7 +47,7 @@
             android:name="twa_generator"
             android:value="@string/generatorApp" />
 
```

Comment:
There has to be an additional application level receiver to be added for capturing app installs correctly with Appsflyer. 

<receiver android:name="com.appsflyer.SingleInstallBroadcastReceiver" android:exported="true">
            <intent-filter>
                <action android:name="com.android.vending.INSTALL_REFERRER" />
            </intent-filter>
        </receiver>

Reply:
Need to add some additional permissions as well - ACCESS_NETWORK_STATE; ACCESS_WIFI_STATE; READ_PHONE_STATE

Reply:
there has to be a new Application class and includes callback code for appsflyer. This new applciation class then has to be referenced something like android:name=".MyApp" inside the application tag within the manifest file.

Reply:
Added all of them. PTAL